### PR TITLE
Implemented the SyncWord Command Functionality

### PR DESCRIPTION
**Pull request by sunnykath**
Moved the SyncWord option up a level and implemented it's functionality.
Now the syncword can be changed throught the command line.

### CODE REVIEW

File: src/util/main.rs [hunk 1]
```diff
@@ -62,6 +62,14 @@ fn main() {
             return
         },
         _ => {
+	    match opts.syncword {
```
Comment (ryankurte) [inline src/util/main.rs:65]: `if let` is a good pattern if you only need one of the match options

```suggestion
	    if let Some(syncword) = opts.syncword {
```

File: src/util/options.rs [hunk 2]
```diff
@@ -58,10 +63,21 @@ pub enum Command {
...
+
+impl std::str::FromStr for HexData {
+    type Err = hex::FromHexError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        hex::decode(s).map(HexData)
+    }
 }
 
+//fn try_from_hex(s: &str) -> Result<u32, std::num::ParseIntError> {
```
Comment (ryankurte) [inline src/util/options.rs:77]: `FromStr` is a good option too, you can remove this?

File: src/util/options.rs [hunk 3]
```diff
@@ -37,6 +37,11 @@ pub struct Options {
     #[structopt(long, default_value = "info")]
     /// Configure radio log level
     pub log_level: LevelFilter,
+
+    /// Set sync word in base Hex (from LSB to MSB without spaces)
```
Comment (ryankurte) [inline src/util/options.rs:41]:
```suggestion
    /// Set sync word in hex (base 16), from LSB to MSB without spaces
```

**Comment by sunnykath** [comment]
Added the recommended changes :) 👍🏽

**Pull request merged**